addons/hr_holidays/static/src/bugfix/bugfix.js

```diff
@@ -8,3 +8,77 @@ odoo.define('hr_holidays/static/src/bugfix/bugfix.js', function (require) {
 'use strict';
 
 });
+
+// FIXME move me in hr_holidays/static/src/models/partner/partner.js
+odoo.define('hr_holidays/static/src/models/partner/partner.js', function (require) {
+'use strict';
+
+const {
+    registerClassPatchModel,
+    registerFieldPatchModel,
+    registerInstancePatchModel,
+} = require('mail/static/src/model/model_core.js');
+const { attr, one2one } = require('mail/static/src/model/model_field.js');
+
+const { str_to_datetime } = require('web.time');
+
+
+registerClassPatchModel('mail.partner', 'hr_holidays/static/src/models/partner/partner.js', {
+    /**
+     * @override
+     */
+    convertData(data) {
+        const data2 = this._super(data);
+        if ('out_of_office_date_end' in data) {
+            data2.out_of_office_date_end = data.out_of_office_date_end;
```

Comment:
So we just completely ignored this value before the PR? interesting...

Reply:
I think here you'll need to handle `!out_of_office_date_end` -> `clear()`, otherwise you'll have a mix of `false` and string/datetime (?)